lib/parser.dart

```diff
@@ -23,15 +23,17 @@ import 'src/vlq.dart';
 // the string represenation.
 // TODO(tjblasi): Ignore the first line of [jsonMap] if the JSON safety string
 // `)]}'` begins the string representation of the map.
-Mapping parse(String jsonMap, {Map<String, Map> otherMaps, mapUrl}) =>
+Mapping parse(String jsonMap,
+        {Map<String, Map>? otherMaps, /*String|Uri*/ Object? mapUrl}) =>
```

Comment:
We can do this in a followup instead of this PR, heads up that I plan on looking at making these all `Uri` instead of `Object?` since we have the opportunity with a breaking version bump.

https://github.com/dart-lang/source_maps/issues/47

Reply:
That should be fine, but I think a followup is best